shared/cfg/guest-os/Linux/Fedora/30.cfg

```diff
@@ -0,0 +1,49 @@
+- 30:
+    variants:
+        - x86_64:
+            vm_arch_name = x86_64
+        - aarch64:
+            vm_arch_name = aarch64
+    image_name = images/f30-${vm_arch_name}
+    os_variant = fedora30
+    # default boot path is set in ../Fedora.cfg: boot_path = "images/pxeboot"
+    no unattended_install..floppy_ks
+    unattended_install, svirt_install:
+        kernel = images/f30-${vm_arch_name}/vmlinuz
+        initrd = images/f30-${vm_arch_name}/initrd.img
+        # Unattended-file does not require any changes
+        unattended_file = unattended/Fedora.ks
+        unattended_install.url:
+            # Installation works fine with mem=1024 on methods such as cdrom
+            # but fails ("No space left on device") with methods such as url.
+            mem = 2048
+            # Unless overridden use secondary url, because it's the most common one
+            url = https://download.fedoraproject.org/pub/fedora/linux/releases/30/Server/${vm_arch_name}/os/
+        x86_64:
+            kernel_params = "console=tty0 console=ttyS0"
+            unattended_install.cdrom:
+                cdrom_cd1 = isos/linux/Fedora-Server-dvd-x86_64-30-1.2.iso
+                md5sum_cd1 = a24d2fed6440df75bd49d531f71a2b84
+                md5sum_1m_cd1 = 07c58a0139b02d178ed2e88eb20f5ca7
+            unattended_install.url:
+                sha1sum_vmlinuz = 89f9019a03b49a2c8a4aa8617710e3e1563dc6d6
+                sha1sum_initrd = b3451ed1214c284fca180e0678b0c46549e5325e
+        aarch64:
+            unattended_install, svirt_install:
+                unattended_file = unattended/Fedora.ks
+            kernel_params = "console=ttyAMA0 console=ttyS0 serial"
+            unattended_install.cdrom:
+                cdrom_cd1 = isos/linux/Fedora-Server-dvd-aarch64-30-1.2.iso
+                md5sum_cd1 = 171fc8c60f08beba41b3d83c6ebda1b0
+                md5sum_1m_cd1 = a14b71cb518f2e5a1717ca2daf3c6dfa
+            unattended_install.url:
+                sha1sum_vmlinuz = bf1247d61743575f03f8e8fcbb9aa8335b1e72b1
+                sha1sum_initrd = 227b9756fd458a4bf877e2653d23c1bc66a28f51
```

Comment:
it'd be nice to introduce all archs at the same time. It's not much extra work, but people testing the other archs will have something to use. We have to add it for JeOS anyway.

Reply:
(and the F28 config already contains all the bits to split between secondary and primary urls...)

Reply:
btw let me know if you want me to build JeOS for the alt archs (not sure whether s390x installs in TCG mode...)